### DISCUSSION TITLE
How can I recover deleted photos in Google Photos?

### DISCUSSION THREAD

**Discussion by lucas-0liveira** [simulot/immich-go#27] [Q&A] [answered]
I'm experiencing an issue trying to retrieve photos that were accidentally deleted in Google Photos. Is there an effective method to restore these images? Thank you in advance for any help or guidance provided!

**Comment by simulot** [comment, marked as answer]
This is a question for google team...

But if you have simply trashed them, not long ago, and if you haven't emptied the trashcan,  you can reopen the trashcan using this link https://photos.google.com/trash

If you have shared you photo with a partner, you can user his/her account to downloads a takeout archive.
I hope you will retrieve your photos.
